src/widgets/DataTable/DataTableRow.js

```diff
@@ -87,11 +87,14 @@ const DataTableRow = React.memo(
     const { isSelected = false } = rowState || {};
     // If the row is selected, use selectedRow style, otherwise alternate row style on index.
 
-    const rowStyle = {
-      ...(rowIndex % 2 === 0 ? alternateRowStyle : basicRowStyle),
-      ...(isSelected && selectedRowStyle),
-      ...(!isValidated && { borderColor: SUSSOL_ORANGE, borderWidth: 1, borderRadius: 5 }),
-    };
+    const rowStyle = useMemo(
+      () => ({
+        ...(rowIndex % 2 === 0 ? alternateRowStyle : basicRowStyle),
+        ...(isSelected && selectedRowStyle),
+        ...(!isValidated && { borderColor: SUSSOL_ORANGE, borderWidth: 1, borderRadius: 5 }),
+      }),
+      [rowIndex, isSelected, isValidated]
+    );
```

Comment:
Could you squeeze a bit more performance here but separating out into individual `useMemo`s for different use cases, e.g. `rowIndex` for sorting, `isSelected` for selecting, etc.?

Reply:
I think only for when the row index changes and the `% 2` result is the same? Don't think it's worth it

Reply:
I mean't more that you could make it so that you only needed to worry about the selected row style when `isSelected` changes and the validated styles when `isValidated` changes etc. (rather than calculating them all when any of the variables changes). But it's such a small thing, agree it's not really worth it!